docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/sparse/embedding-bag-offsets-15.rst

```diff
@@ -0,0 +1,112 @@
+.. {#openvino_docs_ops_sparse_EmbeddingBagOffsets_15}
+
+EmbeddingBagOffsets
+======================
+
+
+.. meta::
+  :description: Learn about EmbeddingBagOffsets-15 - a sparse operation, which
+                can be performed on three required and two optional input tensors.
+
+**Versioned name**: *EmbeddingBagOffsets-15*
+
+**Category**: *Sparse*
+
+**Short description**: Computes sums or means of "bags" of embeddings, without instantiating the intermediate embeddings.
+
+**Detailed description**:
+
+Operation EmbeddingBagOffsets is an implementation of ``torch.nn.EmbeddingBag`` with indices and offsets inputs being 1D tensors.
+
+For each index in ``indices`` this operator gathers values from ``emb_table`` embedding table. Indices that are in range of same bag based on ``offset`` input values are then reduced according to ``reduction`` attribute.
+Values in ``offsets`` define starting index in ``indices`` tensor of each "bag",
+e.g. ``offsets`` with value ``[0, 3, 4, 4, 6]`` define 5 "bags" containing ``[3, 1, 0, 2, num_indices-6]`` elements corresponding to ``[indices[0:3], indices[3:4], empty_bag, indices[4:6], indices[6:]]`` slices of indices per bag.
+
+**Attributes**:
+
+* *reduction*
+
+  * **Description**: reduction mode. 
+  * **Range of values**:
+
+    * sum - compute weighted sum, using corresponding values of ``per_sample_weights`` as weights if provided.
+    * mean - compute average of values in bag. Input ``per_sample_weights`` is not supported and will raise exception.
+
+  * **Type**: ``string``
+  * **Default value**: sum
+  * **Required**: *no*
+
+**Inputs**:
+
+* **1**: ``emb_table`` tensor containing the embedding lookup table of the module of shape ``[num_emb, emb_dim1, emb_dim2, ...]`` and  of type *T*. **Required.**
+* **2**: ``indices`` tensor of shape ``[num_indices]`` and of type *T_IND*. **Required.**
+* **3**: ``offsets`` tensor of shape ``[batch]`` and of type *T_IND* containing the starting index positions of each "bag" in ``indices``. Maximum value of offsets cannot be greater than length of ``indices``. **Required.**
+* **4**: ``default_index`` scalar of type *T_IND* containing default index in embedding table to fill empty "bags". If set to ``-1`` or not provided, empty "bags" are filled with zeros. Reverse indexing using negative values is not supported. **Optional.**
```

Comment:
Usage of `-1` is not an ideal solution, but as we discussed it's aligned with current behavior of some plugins and it resolves the empty bags issues. Although the main reason of adding v15 version was enablement of the "mean" mode, great to see the clarification of the `default_index` behavior as well.